config/initializers/chartkick.rb

```diff
@@ -66,7 +65,7 @@
    plotOptions: {
       series: {
          dataLabels: {
-            color: '#0088CC'
+            color: '#000'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.